pkgs/applications/misc/syncthingtray/default.nix

```diff
@@ -0,0 +1,50 @@
+{ mkDerivation, stdenv, lib, fetchFromGitHub
+, qtbase, extra-cmake-modules, cpp-utilities, qtutilities
+, cmake, kio, plasma-framework, qttools
+, webviewProvider ? null
+, jsProvider ? null
+, enableKioPluginSupport ? false
+, enablePlasmoidSupport  ? false
+, systemdSupport ? true }:
+
+mkDerivation rec {
+  version = "0.9.1";
+  pname = "syncthingtray";
+
+  src = fetchFromGitHub {
+    owner  = "Martchus";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "0ijwpwlwwbfh9fdfbwz6dgi6hpmaav2jm56mzxm6as50iwnb59fx";
+  };
+
+  buildInputs = [ qtbase cpp-utilities qtutilities webviewProvider jsProvider ] 
+    ++ lib.optional enableKioPluginSupport kio
+    ++ lib.optional enablePlasmoidSupport [ extra-cmake-modules plasma-framework ]
+  ;
+
+  nativeBuildInputs = [ cmake qttools ];
+
+  cmakeFlags = lib.optionals (enablePlasmoidSupport == false) ["-DNO_PLASMOID=ON"]
+    ++ lib.optionals (enableKioPluginSupport == false) ["-DNO_FILE_ITEM_ACTION_PLUGIN=ON"]
+    ++ lib.optionals systemdSupport ["-DSYSTEMD_SUPPORT=ON"]
+  ;
+  # Without this hook, `make install` fails since it tries to write to ${qtbase.out}/lib/qt-<version>/...
```

Comment:
I think we should bother the author in this case. When `make install` just ignores the prefix and tries to put files in the system qt directory unconditionally, that looks like a bug. We can merge this workaround anyway, but I'd prefer it if you at least open an issue upstream and link to it in the comment.

Reply:
I thought about it but usually non NixOS developers are not familiar with the concept of per-package prefix and it's kind of hard to explain it to them in contexts such as this. I've opened an issue upstream never the less: https://github.com/Martchus/syncthingtray/issues/42